lib/server.js

```diff
@@ -159,11 +161,11 @@ WalletService.getInstanceWithAuth = function(opts, cb) {
   var server = new WalletService();
   server.storage.fetchCopayerLookup(opts.copayerId, function(err, copayer) {
     if (err) return cb(err);
-    if (!copayer) return cb(new ClientError('NOTAUTHORIZED', 'Copayer not found'));
+    if (!copayer) return cb(new ClientError(Errors.codes.NOT_AUTHORIZED, 'Copayer not found'));
```

Comment:
Glad you bring this up. The `getInstanceWithAuth` method could return a `NOT_AUTHORIZED` error based on two conditions: `Copayer not found` and `Invalid signature`. Since this is a rather special code (see https://github.com/bitpay/bitcore-wallet-service/blob/master/lib/expressapp.js#L72) I was hesitant to create two separate errors for this (e.g. `COPAYER_NOT_FOUND` & `INVALID_SIGNATURE`) which would have been more in line with the rest of the code.
Do you have suggestions to improve this and make error handling less complex?